package.json

```diff
@@ -39,19 +39,22 @@
     "typescript": "^5.1.6"
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^6.12.0",
-    "@typescript-eslint/parser": "^6.12.0",
-    "eslint-config-prettier": "^9.0.0",
+    "@typescript-eslint/eslint-plugin": "^6.19.1",
+    "@typescript-eslint/parser": "^6.19.1",
+    "eslint-config-prettier": "^9.1.0",
     "eslint-import-resolver-typescript": "^3.6.1",
-    "eslint-plugin-import": "^2.29.0",
+    "eslint-plugin-import": "^2.29.1",
     "eslint-plugin-no-only-tests": "^3.1.0",
     "eslint-plugin-prefer-arrow": "^1.2.3",
     "eslint-plugin-react": "^7.33.2",
-    "eslint-plugin-unicorn": "^49.0.0"
+    "eslint-plugin-unicorn": "^50.0.1"
   },
   "config": {
     "commitizen": {
       "path": "./node_modules/mol-conventional-changelog"
     }
+  },
+  "engines": {
+    "node": "20.x"
```

Comment:
is this in the foreseeable future 😄 ..., why not using something more loose like:
`"node": ">= 14.x"`